shared/i18n/locales/fr_FR/translation.json

```diff
@@ -433,12 +433,12 @@
   "Publishing": "Publication en cours",
   "No history yet": "Pas encore d'historique",
   "Stats": "Statistiques",
-  "{{ count }} minute read": "{{ count }} minute read",
-  "{{ count }} minute read_plural": "{{ count }} minute read",
+  "{{ count }} minute read": "Durée de lecture de {{ count }} minute",
+  "{{ count }} minute read_plural": "Durée de lecture de {{ count }} minutes",
```

Comment:
there is a typo here, it should be "minutes"

Reply:
Do you mean in the key name? Those are kept singular as part of the internationalization framework

Reply:
Ok, but in the crowndin UI both strings (singular and plurial) appears as the same, and there is no hint the second was for the plural form.
![image](https://user-images.githubusercontent.com/80455946/204078593-48a5db8c-50ff-47ab-a3cc-0d5871bc6d49.png)

Reply:
Ah weird, I suppose it's because it's the same in english.